tests/core/test_coring.py

```diff
@@ -4122,6 +4122,18 @@ def test_salter():
     with pytest.raises(ShortageError):
         salter = Salter(qb64='')
 
+    salter = Salter(raw=raw)
+    assert salter.stretch(temp=True) == b'\xd4@\xeb\xa6x\x86\xdf\x93\xd6C\xdc\xb8\xa6\x9b\x02\xafh\xc1m(L\xd6\xf6\x86YU>$[\xf9\xef\xc0'
+
+    salter = Salter(raw=raw)
+    assert salter.stretch(tier=Tiers.low) == b'\xf8e\x80\xbaX\x08\xb9\xba\xc6\x1e\x84\r\x1d\xac\xa7\\\x82Wc@`\x13\xfd\x024t\x8ct\xd3\x01\x19\xe9'
+
+    salter = Salter(raw=raw)
+    assert salter.stretch(tier=Tiers.med) == b',\xf3\x8c\xbb\xe9)\nSQ\xec\xad\x8c9?\xaf\xb8\xb0\xb3\xcdB\xda\xd8\xb6\xf7\r\xf6D}Z\xb9Y\x16'
+
+    salter = Salter(raw=raw)
+    assert salter.stretch(tier=Tiers.high) == b'(\xcd\xc4\xb85\xcd\xe8:\xfc\x00\x8b\xfd\xa6\tj.y\x98\x0b\x04\x1c\xe3hBc!I\xe49K\x16-'
```

Comment:
Re-reading this it's a bit ridiculous to instantiate the salter 4 times, I'll change that